app/prj.conf

```diff
@@ -202,6 +214,7 @@ CONFIG_APP_LED_LOG_LEVEL_DBG=y
 CONFIG_APP_ENVIRONMENTAL_LOG_LEVEL_DBG=y
 CONFIG_APP_SHELL_LOG_LEVEL_DBG=y
 CONFIG_APP_LOG_LEVEL_DBG=y
+CONFIG_LOCATION_LOG_LEVEL_DBG=y
```

Comment:
Was it intentional to leave this turned on?

Reply:
I don't recall. Probably not :cry: